src/Utility.ts

```diff
@@ -15,11 +15,10 @@ export class Utility {
   }
 
   /**
-   * Sets the Azure CLI to dynamically install extensions that are missing. In this case, we care about the
-   * Azure Container Apps module being dynamically installed while it's still in preview.
+   * Sets the Azure CLI to dynamically install extensions that are missing.
    */
   public async setAzureCliDynamicInstall() {
```

Comment:
Nit: not for this PR, but this method name isn't super meaningful anymore

Reply:
> Thanks Dan! I've made a note of this.